src/vscode-bicep/schemas/bicepconfig.schema.json

```
@@ -735,7 +734,8 @@
           "type": "boolean"
         },
         "publishSource": {
-          "type": "boolean"
+          "type": "boolean",
+          "description": "Enables publishing source code with modules using the bicep publish `--with-source` option. See https://github.com/Azure/bicep/blob/main/docs/experimental-features.md#publishsource"
```

FYI, we have an aka.ms link
```suggestion
          "description": "Enables publishing source code with modules using the bicep publish `--with-source` option. See https://aka.ms/bicep/experimental-features#publishsource"
```